### PR TITLE
Setup /etc/hosts for containers

### PR DESCRIPTION
**Pull request by feiskyer**
Update `/etc/hosts` volume for containers.

### CODE REVIEW

File: hypervisor/devicemap.go [hunk 1]
```diff
@@ -612,6 +613,71 @@ func (ctx *VmContext) ReleaseInterface(index int, ipAddr string, file *os.File,
...
+	if volumeName == "" {
+		return
+	}
+
+	for _, vol := range ctx.userSpec.Volumes {
+		if vol.Name == volumeName && vol.Driver == "vfs" {
+			volumePath = vol.Source
+		}
+	}
+
```
Comment (gnawux) [inline hypervisor/devicemap.go]: How can you tell if the file is specified by the user or generated by hyperd?

Comment (feiskyer) [inline hypervisor/devicemap.go]: We should add `ip hostname` to hosts for both cases.

**Comment by laijs** [comment]
close it. please reopen it when needed.

**Comment by feiskyer** [comment]
Why this is closed?

**Comment by feiskyer** [comment]
I'm closing this since this could be resolved by external dns.

**Pull request closed without merging**